src/blocks/MusicBlock.tsx

```diff
@@ -0,0 +1,106 @@
+import Block from './Block';
+import { BlockModel } from './types';
+import './BlockStyles.css';
+import { Stack, Box } from '@mui/material';
+import { useState, useEffect, useRef } from 'react';
+import SeamSaveButton from '../components/SeamSaveButton';
+import Iframely from './utils/Iframely';
+
+interface Music {}
+
+const SearchScreen: React.FC<any> = ({ done, model, onFinalize }) => {
+  const [url, setUrl] = useState<string>(model.data.url || '');
+  const [SpotifySearchBar, setSpotifySearchBar] = useState<any>(null);
+  const isMounted = useRef(true);
+
+  useEffect(() => {
+    if (process.env.REACT_APP_IS_SUPERAPP === 'true') {
+      const loadSpotifySearchBar = () => {
+        try {
+          const module = require('../../../../src/SpotifySearch/SpotifySearchBar').default;
```

Comment:
don't start the pattern of allowing miniapps to pull code in from our superapp, this is a strong antipattern which violates the sandboxing of miniapps. instead, if a miniapp needs the code, put it in as a component into the miniapp repo.